src/PointwiseFunctions/Hydro/Python/ComovingMagneticField.hpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+namespace py_bindings {
```

Comment:
Should this be `hydro::py_bindings`? @nilsvu what's our convention here?

Reply:
Yep, `hydro::pybindings`